pymc/distributions/multivariate.py

```diff
@@ -1868,7 +1884,7 @@ def rng_fn(self, rng, mu, sigma, *covs, size=None):
         if sigma:
             cov = cov + sigma**2 * np.eye(cov.shape[0])
 
-        x = multivariate_normal.rng_fn(rng=rng, mean=mu, cov=cov, size=size)
+        x = pymc_multivariate_normal.rng_fn(rng=rng, mean=mu, cov=cov, size=size)
```

Comment:
No need to change this or above. Only thing that matters is the `rv_op` we use in the distribution classes. All other changes should be undone.